lte/gateway/python/mypy.ini

```diff
@@ -0,0 +1,19 @@
+[mypy]
+python_version = 3.8
+ignore_missing_imports = True
+show_error_codes = True
+warn_unused_configs = True
+ignore_errors = False
```

Comment:
Could we add an option `namespace_packages = True` here to deal with packages without the `__init__.py`.

Reply:
You would probably need to adopt the `    ^python/magma/subscriberdb/protocols/diameter/ |` line and remove the `diameter` subfolder from it, though.

Reply:
And maybe also the
```
install_types = True
non_interactive = True
```

Reply:
As discussed yesterday I moved the flags into the `mypy.ini`